### ISSUE TITLE
Lots of unexpected white

### ISSUE THREAD

**Issue by WGINC** [Gameslinx/Tessellation#41]
Whelp, here is my problem.

<img width="1440" alt="Screen Shot 2021-02-27 at 10 29 42 PM" src="https://user-images.githubusercontent.com/77513369/109407152-8a516e80-794c-11eb-912c-5c5f44d28b6f.png">
<img width="1440" alt="Screen Shot 2021-02-27 at 10 30 46 PM" src="https://user-images.githubusercontent.com/77513369/109407157-8e7d8c00-794c-11eb-9f21-c9e36a35da6a.png">
<img width="1440" alt="Screen Shot 2021-02-27 at 10 30 56 PM" src="https://user-images.githubusercontent.com/77513369/109407166-9a694e00-794c-11eb-8e66-c6b99c7ad6c4.png">
<img width="1440" alt="Screen Shot 2021-02-27 at 10 32 39 PM" src="https://user-images.githubusercontent.com/77513369/109407172-a523e300-794c-11eb-8113-ee127c025cde.png">
<img width="1440" alt="Screen Shot 2021-02-27 at 10 34 05 PM" src="https://user-images.githubusercontent.com/77513369/109407175-aa812d80-794c-11eb-97dd-cb58773dacfb.png">
<img width="1440" alt="Screen Shot 2021-02-27 at 10 35 00 PM" src="https://user-images.githubusercontent.com/77513369/109407177-b0770e80-794c-11eb-9807-56f4b46d8adb.png">

In summary, lots of white.

MacOSX Big Sur 11.2.1
KSP 1.11.1
Parallax 1.2.1, Parallax stock textures 1.2.1, Kopernicus Bleeding edge UBE-80
No other visual enhancements mods
[KSP.log](https://github.com/Gameslinx/Tessellation/files/6056238/KSP.log)

**Comment by EmeraldWither** [comment]
Try using the Stable version of Kopernicus here : https://github.com/Kopernicus/Kopernicus.

**Comment by kailGH** [comment]
R34 is the stable version of Kopernicus, it was the second thing I tried.

**Comment by ondrejchmelar** [comment]
IIRC this happened when updating Parallax by extracting over the existing files. Try removing the plugin dirs first.

**Comment by kailGH** [comment]
I'll give that a try when able. Although I didn't actually do the update manually, CKAN did.

**Comment by EmeraldWither** [comment]
That might be the problem, try to install it manually.

**Comment by WGINC** [comment]
I installed both mods (Kopernicus(R34 and UBE-80)/Parallax) manually, EmeraldWither. And how do I remove plugin dirs.

**Comment by EmeraldWither** [comment]
> I installed both mods (Kopernicus(R34 and UBE-80)/Parallax) manually, EmeraldWither. And how do I remove plugin dirs.

Make sure to completely delete all of the folders related to Parallax, and then reinstall. Make sure that you also have the stock textures installed. I would also try re-verifying the integrity of files on steam if you can.

**Comment by WGINC** [comment]
Ok, will do.

**Comment by EmeraldWither** [comment]
Also make sure that your terrain quality is set to high.

**Comment by WGINC** [comment]
ok

**Comment by WGINC** [comment]
<img width="1026" alt="Screen Shot 2021-03-02 at 9 05 34 PM" src="https://user-images.githubusercontent.com/77513369/109741480-25d81e80-7b9b-11eb-9859-5cd2fa8ff95f.png">
<img width="1032" alt="Screen Shot 2021-03-02 at 9 04 42 PM" src="https://user-images.githubusercontent.com/77513369/109741491-2c669600-7b9b-11eb-9927-7160270409c2.png">

**Comment by EmeraldWither** [comment]
Yikess, uh, try to make sure that you are using a normal version of the game without mods.

**Comment by victormakeaveli** [comment]
Try this: with a clean new install of ksp, [1]manually install kopernicus again. I mean.. it worked for me.
If you dont wanna do a clean ksp install again try completely removing kopernicus and then[1].
For the latest version of Parallax I'd recommend also the lastest kop:  KopernicusBE_19+_Release80.

**Comment by Gameslinx** [comment]
You do not meet the following minimum system requirements: **Shader Model 5.0,  GT 960M**
You **cannot** run the Parallax shader without being on Shader Model 5.0 - You are using Shader Model 4.6.
You need a dedicated GPU to run this mod
